### PR TITLE
Fixed Volume Limit Issue

### PR DESCRIPTION
**Pull request by Kakarot-0690**
Volume Could Be Set At 1000, just added a simple if, else statement to avoid that

**Changed Lines from 28 - 34:**

**Max Volume Can Only Be Set Till 100**

### CODE REVIEW

**Comment by SudhanPlayz** [comment]
I still don't know why is this needed

**Comment by Kakarot-0690** [comment]
You can literally set volume at 1000 which breaks audio so there is a necessity for a limit

File: commands/volume.js [hunk 1]
```diff
@@ -25,8 +25,13 @@ module.exports = {
         if (message.guild.me.voice.channel && message.member.voice.channel.id !== message.guild.me.voice.channel.id) return client.sendTime(message.channel, ":x: | **You must be in the same voice channel as me to use this command!**");
         if (!parseInt(args[0])) return client.sendTime(message.channel, `**Please choose a number between** \`1 - 100\``);
         let vol = parseInt(args[0]);
+        if(vol>0 && vol>101){
```
Comment (SudhanPlayz) [inline commands/volume.js:28]:
```suggestion
        if(vol < 0 || vol > 100){
```

File: commands/volume.js [hunk 2]
```diff
@@ -25,8 +25,13 @@ module.exports = {
         if (message.guild.me.voice.channel && message.member.voice.channel.id !== message.guild.me.voice.channel.id) return client.sendTime(message.channel, ":x: | **You must be in the same voice channel as me to use this command!**");
         if (!parseInt(args[0])) return client.sendTime(message.channel, `**Please choose a number between** \`1 - 100\``);
```
Comment (SudhanPlayz) [inline commands/volume.js:26]:
```suggestion
        if (!parseInt(args[0])) return client.sendTime(message.channel, `Please choose a number between \`1 - 100\``);
```

Comment (SudhanPlayz) [reply commands/volume.js:26]: maybe not so important

File: commands/volume.js (hunk 1 above)
Comment (SudhanPlayz) [reply commands/volume.js:28]: Check this code if its work let me know so i can commit

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Check the code is it working fine or doesn't

File: commands/volume.js (hunk 1 above)
Comment (Kakarot-0690) [reply commands/volume.js:28]: yes works either way ;-;

File: commands/volume.js (hunk 2 above)
Comment (Kakarot-0690) [reply commands/volume.js:26]: lol

**Comment by Kakarot-0690** [comment]
> Check the code is it working fine or doesn't

works fine, u can commit

**Comment by DarrenOfficial** [comment]
what if sudhan doesnt wanna commit ;-;

**Comment by Kakarot-0690** [comment]
> what if sudhan doesnt wanna commit ;-;

his bot, his wish lmfao, I just saw a random issue so thought would contribute

**Comment by DarrenOfficial** [comment]
> his bot, his wish lmfao, I just saw a random issue so thought would contribute

understandable, have a great day!

**Comment by Kakarot-0690** [comment]
> > his bot, his wish lmfao, I just saw a random issue so thought would contribute
> 
> understandable, have a great day!

*have a great day with me

**Pull request merged**